cmd/control/cmd/status.go

```diff
@@ -24,7 +24,9 @@ var statusCmd = &cobra.Command{
 		fmt.Println(" * installed: " + version + " (" + commit + ")")
```

Comment:
It would be nice to print out the version of resh in the session even when the resh-daemon is not running.
Could you move the code for showing version of "this session" here - right after printing "installed" version.
https://github.com/curusarn/resh/blob/88f503f27fd0788f8868a02937aed104c80656c4/cmd/control/cmd/status.go#L33-L41